src/sentry/tasks/derive_code_mappings.py

```diff
@@ -17,42 +24,114 @@
 logger = logging.getLogger("sentry.tasks.derive_code_mappings")
 
 
+@instrumented_task(
+    name="sentry.tasks.derive_code_mappings.process_organizations",
+    queue="derive_code_mappings",
+    max_retries=0,
+)  # type: ignore
+def derive_missing_codemappings(dry_run=False) -> None:
+    """
+    Queue tasks to process each organization to derive missing codemappings.
+    """
+    organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
+    for _, organization in enumerate(
+        RangeQuerySetWrapper(organizations, step=1000, result_value_getter=lambda item: item.id)
+    ):
+        if not features.has("organizations:derive-code-mappings", organization):
```

Comment:
Do we have an existing feature flag in getsentry?

Reply:
We do now.